package.json

```diff
@@ -10,6 +10,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@mui/material": "^5.15.21",
```

Comment:
Let's remove Materials UI since we plan to create our Styled Components using Emotion. See note about how to do that here: https://github.com/unocorn-squad/job-posting-app?tab=readme-ov-file#creating-styled-components-using-emotion

Reply:
I am working on this.